gdeployfeatures/lv/lv.py

```diff
@@ -24,11 +30,17 @@ def lv_create(section_dict):
          section_dict, yml = get_lv_vg_names('poolname', section_dict)
     else:
         print "Error: Unknown lvtype"
+        if Global.trace:
+            Global.logger.info("Retrieved lv type unknown.")
```

Comment:
This is not info, this is error. You have to log error saying "Unknown lv type". You cannot say "Retrieved lv type unknown"